exercises/G-numbers/exercise.js

```diff
@@ -1 +1,10 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = "Number of students: 15"
```

Comment:
No way, you rigged here. I think code should have been like, <<"Number of students: " + numberOfStudents

Reply:
Noted, will take care of it.

Reply:
Done, thanks for the feedback.